MdePkg/Library/BaseRngLib/BaseRngLibInternals.h

```diff
@@ -11,6 +11,8 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #ifndef BASE_RNGLIB_INTERNALS_H_
 #define BASE_RNGLIB_INTERNALS_H_
 
+#include <AArch64/AArch64.h>

```

Comment:
can't this include be moved to MdePkg/Library/BaseRngLib/AArch64/ArmRng.h

Reply:
I can move it there. The ASM file that uses it only includes this file: https://github.com/tianocore/edk2/blob/2d2642f4832ebc45cb7d5ba9430b933d953b94f2/MdePkg/Library/BaseRngLib/AArch64/ArmRng.S#L11. But I agree that having it in the ARM64 specific header is better.